### PR TITLE
lesson5

### PR DESCRIPTION
**Pull request by AZhurav1ev**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -317,12 +325,72 @@ const treeTable = {
...
+            elements: [
+                {
+                    view: "text",
+                    id: "category_input",
+                    placeholder: "Category name",
+                },
+                {
+                    view: "button",
+                    label: "Add Category",
+                    id: "btn_add_category",
```
Comment (Zwillinge) [inline script.js]: don't set widget id if you do not plan to use it (same for delete button)

File: script.js [hunk 2]
```diff
@@ -317,12 +325,72 @@ const treeTable = {
...
+                            webix.message(`Category deleted`);
+                        }
+                    }
+                },
+            ]
+        },
+        {
+            view: "datatable",
+            id: "categoriesTable",
+            autoConfig: true,
```
Comment (Zwillinge) [inline script.js]: don't use both autoConfig and columns, as they contradict each other

**Comment by Zwillinge** [comment]
Also, please fix some UX problems: 

- preselect the first menu item on init, so that users can understand which multiview cell is active now
- provide `fillspace:true` to at least one treetable column to avoid UI jumping when switching between cells

**Comment by Zwillinge** [comment]
Also, you've lost a part of list functionality, scheme for editlist is no longer working, and the users are  not highlighted. The reason is only master Datastore can be processed by a scheme, so you need to move the $init code to the DataCollection.

File: script.js (hunk 1 above)
Comment (AZhurav1ev) [reply script.js]: Thank you, will keep it in mind.

File: script.js (hunk 2 above)
Comment (AZhurav1ev) [reply script.js]: Will keep it in mind too.

**Comment by AZhurav1ev** [comment]
> Also, please fix some UX problems:
> 
> * preselect the first menu item on init, so that users can understand which multiview cell is active now
> * provide `fillspace:true` to at least one treetable column to avoid UI jumping when switching between cells

I know about it, but didn't know how to fix it. Tried to create function in sync method like made it for chart. Here I have a question. How in that case we can apply scheme only for List, but not for both components. For example if we had two list components instead one.

**Comment by AZhurav1ev** [comment]
> Also, you've lost a part of list functionality, scheme for editlist is no longer working, and the users are not highlighted. The reason is only master Datastore can be processed by a scheme, so you need to move the $init code to the DataCollection.

I know about it, but didn't know how to fix it. Tried to create function in sync method like made it for chart. Here I have a question. How in that case we can apply scheme only for List, but not for both components. For example if we had two list components instead one.

**Pull request merged**